### ISSUE TITLE
Show all expanded arguments by default. 

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/wporg-developer#133]
Parameter arrays should not be collapsed/hidden.

Issue Source: https://make.wordpress.org/meta/2022/07/01/exploration-improving-devhub/#comment-9430

**Comment by theMikeD** [comment]
With respect, this ticket seems backwards to me. The items being obscured on a development reference page are not secondary, they are part of the point of the page: a technical reference. The decision to hide them is the change, and should be discussed somewhere before implementation. Instead this seems to be set up as a random change hiding technical developer information, and then a ticket to revert it back to clarity. I don't understand this.

Can you link to the discussion around why this change was made?

**Comment by StevenDufresne** [comment]
> The decision to hide them is the change, and should be discussed somewhere before implementation. Instead this seems to be set up as a random change hiding technical developer information, and then a ticket to revert it back to clarity. I don't understand this.

> Can you link to the discussion around why this change was made?

Incorrect, these properties currently do not exist and are a new addition. Properties in the parameters sections are still appearing as they did before.

**Comment by StevenDufresne** [comment]
A fair criticism of this would be, either:

## 1. We should leave it as if and not display the arguments: 
**Current Implementation:**
<img src="https://d.pr/i/jhsefR.png" />

## 2. We should display the arguments but not have them in ~~a dropdown~~ an accordion.

On second review I would agree that the wording of this issue could be improved as it kind of misses the point:

`Currently, expanded arguments are obscured by an accordion feature because of their potential length and secondary importance to the current page`

Maybe something like:

`Parameter arrays should not be collapsed by default.`

**Comment by theMikeD** [comment]
Fair enough. #2 is what I was getting at. If the properties are important enough to include, they are important enough not to be arbitrarily hidden by an accordion
